src/components/dailySuccessTilt.js

```diff
@@ -34,8 +34,11 @@ class DailySuccessTilt extends Component {
   }
 
   getData(date) {
+    if (!date) {
```

Comment:
Je suis pas certain d'aimer ce fix là.. Quand il a pas de date de setté, ça ressemble à ça. 

![image](https://user-images.githubusercontent.com/10834800/48278780-7ea8a980-e41c-11e8-94b0-300252548a09.png)

Reply:
Avec la modif pour attendre le data, ca devrais pas rester though

Reply:
Même si on attend le data, la date reçue peut rester null

Reply:
Mais ok, tu peux laisser ça

Reply:
Dans angleResult on devrait pas render DailyAngleDistribution seulement si this.state.angle est pas null? @GBoucher42

Reply:
y'a pas de state.angle dans angleResults, pas certain de comprendre ta question.. tu veux dire this.state.date?

Reply:
oups..ouais exacte !

Reply:
fair point, j'accepte